CMSIS/Core/Include/cmsis_tiarmclang.h

```diff
@@ -0,0 +1,1510 @@
+/**************************************************************************//**
+ * @file     cmsis_tiarmclang.h
+ * @brief    CMSIS compiler tiarmclang header file
+ * @version  V5.6.0
```

Comment:
This is a new file and version should start at 1.0.0

Reply:
No problem.  I'm assuming, then, that the file version and date reflect the last file modification?  In what way does the version increment when the file is changed?

Reply:
Yes, reason behind having individual file versions and dates is that still people are used to ship copies of cmsis files (instead of relying on official releases of the pack). In order to sort out user feedback we need to specifically know which version if a files a user received. The version shall follow [SemVer](https://semver.org/lang/de/) rules, i.e., increment major part on non-backward compatible changes (should be avoided), minor part for new features, and patch part for bug-fixes only.

Reply:
Do you plan to set this to 1.0.0 for the initially contributed version?

Reply:
My second commit makes this change to V1.0.0 -- is that not visible?